tests/cases/fourslash/codeFixImportNonExportedMember1.ts

```diff
@@ -0,0 +1,29 @@
+/// <reference path='fourslash.ts' />
+// @Filename: /a.ts
+////declare function zoo(): any;
+////export { zoo };
+
+// @Filename: /b.ts
+////declare function foo(): any;
+////function bar(): any;

```

Comment:
I would like to see some tests on overloaded functions as well.

Reply:
Good shout! do you think the export should be on the implemented function or in a named export? 🤔

Reply:
I think a separate export statement (e.g. `export { add }`) would be acceptable for function overloads.